docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3'
+services:
+  postgres:
+    image: postgres
+    container_name: taxman-postgres
+    ports:
+      - 5432:5432
+    restart: always
+    environment:
+      POSTGRES_USER: taxman
+      POSTGRES_DATABASE: taxman
+      POSTGRES_PASSWORD: taxman
+  taxman:
+    build:
+      dockerfile: ./dockerfiles/Dockerfile.jvm
+      context: .
+    links:
+      - postgres
+    ports:
+      - 8104:8080
```

Comment:
Port `8104`? The backend will bind to `8080` on the container by default.

Reply:
No problem, updated that. Force of habit choosing random ports, always run into conflicts.